Norma

```diff
@@ -54,6 +54,7 @@ tasks:
       export:
         "Apollos": "both"
         "HTTP": "server"
+        "moment": "both"
```

Comment:
I'm using it client side, but thought it could be useful on the server as well.  Trying to have some foresight.  You want me to change it?